java/ql/test/experimental/query-tests/security/CWE-275/Test.java

```diff
@@ -0,0 +1 @@
+public class Test{}
```

Comment:
Is this file included on purpose?

Reply:
Yes, XML extractor can only be called from a parent language extractor. In this case, seeing the `Test.java` file, the java extractor is invoked which in turn recognizes `xml` and includes it in the db. Without it, there won't be any xml included in the generated db.

Reply:
Are you sure? It appears https://github.com/github/codeql/tree/master/java/ql/test/experimental/query-tests/security/CWE-548 works without Java file